roles/kalite/tasks/install.yml

```diff
@@ -66,3 +66,14 @@
   with_items:
     - { src: 'kalite-serve.service.j2', dest: '/etc/systemd/system/kalite-serve.service', mode: '0644'}
     - { src: 'kalite.sh.j2', dest: '/etc/profile.d/kalite.sh', mode: '0644'}
+    - { src: 'kalite.config', dest: '/etc/{{ apache_config_dir}}', mode: '0644'}
+
```

Comment:
should that be kalite.conf to match the template file name where kalite.config is?